content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -1,7 +1,1704 @@
+
+<!-------------------------------------------------------------->
+
+## `edit-gem-properties`
+
+Edits the specified Gem's properties by modifying the `gem.json` file. 
+
+### Format
+
+```cmd
+edit-gem-properties [-h] (-gp GEM_PATH | -gn GEM_NAME)
+                        [-gnn GEM_NEW_NAME] [-gd GEM_DISPLAY]
+                        [-go GEM_ORIGIN] [-gt {Code,Tool,Asset}]
+                        [-gs GEM_SUMMARY] [-gi GEM_ICON]
+                        [-gr GEM_REQUIREMENTS]
+                        [-at [ADD_TAGS [ADD_TAGS ...]] | -dt
+                        [REMOVE_TAGS [REMOVE_TAGS ...]] | -rt
+                        [REPLACE_TAGS [REPLACE_TAGS ...]]]
+```
+
+### Usage
+
+Updates the `gem_name` field in the `gem.json` file located at the specified Gem's folder.
+
+```cmd
+o3de.bat edit-gem-properties --gem-path GEM_PATH --gem-new-name GEM_NEW_NAME
+
+// or
+
+o3de.bat edit-gem-properties --gem-name GEM_NAME --gem-new-name GEM_NEW_NAME
+```
+
+### Optional parameters
+
+
+- **`-h, --help`**
+- 
+  Shows the help message.
+
+- **`-gp GEM_PATH, --gem-path GEM_PATH`**
+
+  The path to the Gem.
+
+- **`-gn GEM_NAME, --gem-name GEM_NAME`**
+
+  The name of the Gem.
+
+- **`-at [ADD_TAGS [ADD_TAGS ...]], --add-tags [ADD_TAGS [ADD_TAGS ...]]`**
+
+  Adds tag(s) to the `user_tags` property. To add multiple tags, use a space delimited list (for example, `-at A B C`).
+
+- **`-dt [DELETE_TAGS [DELETE_TAGS ...]], --delete-tags [DELETE_TAGS [DELETE_TAGS ...]]`**
+
+  Removes tag(s) from the `user_tags` property. To delete multiple tags, use a space delimited list (for example, `-dt A B C`).
+
+- **`-rt [REPLACE_TAGS [REPLACE_TAGS ...]], --replace-tags [REPLACE_TAGS [REPLACE_TAGS ...]]`**
+  
+  Replace the `user_tags` property with the specified space delimited list of values.
+
+
+**Gem properties:**
+
+  The following parameters modify the specified Gem's properties.
+
+- **`-gnn GEM_NEW_NAME, --gem-new-name GEM_NEW_NAME`**
+
+  Sets the Gem's name.
+
+- **`-gd GEM_DISPLAY, --gem-display GEM_DISPLAY`**
+
+  Sets the Gem's display name.
+
+- **`-go GEM_ORIGIN, --gem-origin GEM_ORIGIN`**
+
+  Sets the description or URL for the Gem origin (such as the Gem host, repository, or owner).
+
+- **`-gt {Code,Tool,Asset}, --gem-type {Code,Tool,Asset}`**
+
+  Sets the Gem type to either Code, Tool, or Asset.
+
+- **`-gs GEM_SUMMARY, --gem-summary GEM_SUMMARY`**
+
+  Sets the Gem's summary description.
+
+- **`-gi GEM_ICON, --gem-icon GEM_ICON`**
+
+  Sets the path to the Gem's icon resource.
+
+- **`-gr GEM_REQUIREMENTS, --gem-requirements GEM_REQUIREMENTS`**
+
+  Sets the description of the requirements that are needed to use the Gem.
+
+<!-------------------------------------------------------------->
+
+## `sha256`
+
+Creates a hash value for an O3DE object using the SHA-256 secure hash algorithm. This command outputs the specified file path and writes the value to the `sha256` field in the specified JSON file. 
```

Comment:
```suggestion
Creates a hash value for an O3DE object using SHA-256 (Secure Hash Algorithm 256). This command outputs the specified file path and writes the value to the `sha256` field in the specified JSON file.
```
Revised for consistency with the updated usage in the **Commands** section intro.